#!/usr/bin/env python3
PACKAGE = "me5413_world"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed_target", double_t, 1, "Default: 0.5[m/s]", 0.5, 0.1, 1.0)
gen.add("lookahead_distance", double_t, 1, "Default: 1.8", 1.8, 0.1, 10.0)
gen.add("steering_gain", double_t, 1, "Default: 0.8", 0.8, 0, 10.0)

exit(gen.generate(PACKAGE, "pure_persuit_node", "pure_persuit"))